src/components/marketplace/views.tsx

```diff
@@ -159,7 +159,7 @@ const serviceDetails: IServiceDetails = {
   'postgres': {
     image: postgresLogo,
     imageTitle: 'PostgreSQL - Official Logo',
-    name: 'Postgres',
+    name: 'RDS Postgres',
```

Comment:
@paroxp how's that?

Reply:
Add the same for MySQL and we have a deal.

Reply:
BEHOLD!